docs/content/routing/entrypoints.md

```diff
@@ -51,199 +56,414 @@ They define the port which will receive the requests (whether HTTP or TCP).
+        address: ":8888"
+        transport:
+          lifeCycle:
+            requestAcceptGraceTimeout: 42
+            graceTimeOut: 42
+          respondingTimeouts:
+            readTimeout: 42
+            writeTimeout: 42
+            idleTimeout: 42
+        proxyProtocol:
+          insecure: true
+          trustedIPs:
+          - "127.0.0.1"
+          - "192.168.0.1"
+        forwardedHeaders:
+          insecure: true
+          trustedIPs:
+          - "127.0.0.1"
+          - "192.168.0.1"
+    ```
+    
+    ```bash tab="CLI"
+    ## Static configuration
+    --entryPoints.name.address=:8888
+    --entryPoints.name.transport.lifeCycle.requestAcceptGraceTimeout=42
+    --entryPoints.name.transport.lifeCycle.graceTimeOut=42
+    --entryPoints.name.transport.respondingTimeouts.readTimeout=42
+    --entryPoints.name.transport.respondingTimeouts.writeTimeout=42
+    --entryPoints.name.transport.respondingTimeouts.idleTimeout=42
+    --entryPoints.name.proxyProtocol.insecure=true
+    --entryPoints.name.proxyProtocol.trustedIPs="127.0.0.1,192.168.0.1"
+    --entryPoints.name.forwardedHeaders.insecure=true
+    --entryPoints.name.forwardedHeaders.trustedIPs="127.0.0.1,192.168.0.1"
+    ```
 
-If proxyprotocol header parsing is enabled for the entry point, this entry point can accept connections with or without proxyprotocol headers.
+### Forwarded Header
 
-If the proxyprotocol header is passed, then the version is determined automatically.
+You can configure Traefik to trust the forwarded headers information (`X-Forwarded-*`).
 
-??? example "Enabling Proxy Protocol with Trusted IPs" 
+??? info "`forwardedHeaders.trustedIPs`"
+    
+    Trusting Forwarded Headers from specific IPs.
 
     ```toml tab="File (TOML)"
+    ## Static configuration
     [entryPoints]
       [entryPoints.web]
         address = ":80"
     
-        [entryPoints.web.proxyProtocol]
+        [entryPoints.web.forwardedHeaders]
           trustedIPs = ["127.0.0.1/32", "192.168.1.7"]
     ```
     
     ```yaml tab="File (YAML)"
+    ## Static configuration
     entryPoints:
       web:
         address: ":80"
-        proxyProtocol:
+        forwardedHeaders:
           trustedIPs:
           - "127.0.0.1/32"
           - "192.168.1.7"
     ```
     
     ```bash tab="CLI"
+    ## Static configuration
     --entryPoints.web.address=:80
-    --entryPoints.web.proxyProtocol.trustedIPs=127.0.0.1/32,192.168.1.7
+    --entryPoints.web.forwardedHeaders.trustedIPs=127.0.0.1/32,192.168.1.7
     ```
 
-    IPs in `trustedIPs` only will lead to remote client address replacement: Declare load-balancer IPs or CIDR range here.
+??? info "`forwardedHeaders.insecure`"
     
-??? example "Insecure Mode -- Testing Environment Only"
-
-    In a test environments, you can configure Traefik to trust every incoming connection.
-    Doing so, every remote client address will be replaced (`trustedIPs` won't have any effect)
+    Insecure Mode (Always Trusting Forwarded Headers).
```

Comment:
```suggestion
```

Reply:
I prefer to be homogeneous with other element.
The name of the option can be long.